app/scripts/modules/core/src/pipeline/config/triggers/triggers.html

```diff
@@ -18,6 +18,17 @@
       </div>
     </div>
   </page-section>
+  <render-if-feature feature="managedServiceAccounts">
+  <page-section key="roles" label="Pipeline Roles" visible="!pipeline.strategy" no-wrapper="true">
```

Comment:
I think this would be better positioned under `Automated Triggers` below, rather than as its own page section - what do you think? I'd put it at the top of the trigger configs.